### ISSUE TITLE
Question: can it handle "apks" file generated from bundletool?

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [Aefyr/SAI#162]
It is possible to convert aab file into "apks" file using bundletool:

https://developer.android.com/studio/command-line/bundletool

Thing is, this seems to fail on SAI.
Can you please explain why?
It says it has "Split config.arm64_v8a defined multiple times", for example.

I assume that the "apks" format of SAI is a different one than the one of bundletool, but still, it seems it tried to install it.

**Comment by Aefyr** [comment]
Yeah, bundletool's .apks format and SAI's one are different. Although both are ZIP archives

**Comment by AndroidDeveloperLB** [comment]
Is it even possible to do it? Seems like it needs more steps from it.
I close this now because it doesn't seem to be related enough.
Still, I'm curious to know what you think or know about this.

**Comment by MuntashirAkon** [comment]
As I was adding support for `.apks` in my app, I noticed the same thing. I initially thought SAI supports the `.apks` generated by the bundletool, then I realised that it doesn't and that `apks` actually means apk set. There's an [official documentaion page for bundletool](https://developer.android.com/studio/command-line/bundletool) to describe how to use the tool itself but there's no documentation about it's contents. I tried to browse an `.apks` file and found that it has two folders: `splits` and `standalones` (the folders speak for themselves) and a file called `toc.pb` (table of content with protobuff extension? I didn't look at the source code so can't tell but it's a binary file). Anyway, I've also noticed some duplicate splits there but with different file sizes, which is probably why SAI is complaining about duplicate split apks.
